Assets/Javascript/index.js

```diff
@@ -1,6 +1,45 @@
 const MenuBtn = document.getElementById('menu_btn');
 const DisplayMenu = document.getElementById('display_menu');
 const CloseMenuBtn = document.getElementById('close_menu_btn');
+const emailInput = document.getElementById('email_input');
+const announce = document.createElement('div');
+const announceText = document.createElement('p');
+const contactForm = document.getElementById('contact_form');
+const emailRegExpr = /[A-Z]+/;
+
+function formValidate() {
+  const emailValidator = emailRegExpr.test(emailInput.value);
+  if (emailValidator) {
+    if (announce) {
+      announce.remove();
+      contactForm.appendChild(announce);
+      announce.appendChild(announceText);
+      announce.className = 'announce_bad_email';
+      announceText.className = 'announce_text';
+      announceText.innerHTML = 'Email address should not contain upercase characters!';
+      emailInput.style = 'animation: email_input_error 0.3s;';
+      setTimeout(() => { emailInput.style = ''; }, 480);
+      return false;
+    }
+    contactForm.appendChild(announce);
+    announce.appendChild(announceText);
+    announce.className = 'announce_bad_email';
+    announceText.className = 'announce_text';
+    announceText.innerHTML = 'Email address should not contain upercase characters!';
+    emailInput.style = 'animation: email_input_error 0.3s;';
+    setTimeout(() => { emailInput.style = ''; }, 480);
+    return false;
+  }
+  return true;
+}
+if (emailRegExpr < 0) {
+  formValidate();
+}
+
+emailInput.addEventListener('input', (event) => {
+  event.preventDefault();
+  announce.remove();
```

Comment:
- [ ] The contact form should contain at least one validation checking if the email is written in all lowercase:

- [ ] If a user submits the contact form with the email field containing uppercase characters an error message should show up.


- [ ] The error message should show up close to the "submit" button.
- [ ] The form "submit" or the submit button/input "click" event is used to attach the function checking the email format.
 - [ ] There should be a check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: email === email.toLowerCase(), utilizing regular expressions, or looping through each letter of the email string. Any implementation of this check should be considered valid as long as it works properly.

Reply:
Hello! Thank you , I really apreciate the time you spent in this code review, but, can you be more specific please?

In response to the first point.- I've actually made an algorihtm that tests the upper and lower case sensitivity to hold form action with a regular expression comparisson, isn't it valid?.

In response to the second point.- A message pop-up with error is shown when user uses uppercase character at the e-mail input.

In response to the third point.- ok, my message is shown near to the e-mail input, so I'll correct that.

In response to the fourth point.- the submit button is actually triggering the validation function set at the javascript index.js file.

In response to the fifth point.- ok I'll do the code for showing the check. Thanks.

Could you please explain to me